### PR TITLE
MAISTRA-2399 Move doc generation from operator to api repo

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Comment by rcernich** [comment]
Do we need to remove the docs files, perhaps adding a link to the new location?

**Comment by brian-avery** [comment]
+1. Docs in this repo should probably be removed. Other than that, this looks good to me.

**Comment by luksa** [comment]
I thought I had removed them, but apparently not. I've now added a link to maistra/api docs to the README.md file.

**Review by rcernich — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -142,6 +142,10 @@ bash <(curl -L https://kiali.io/getLatestKialiOperator) --uninstall-mode true --
 
 For more details on uninstalling the Kiali operator, see the [Kiali documentaton](https://www.kiali.io/documentation/getting-started/#_uninstall_kiali_operator_and_kiali).
 
+## API documentation
+
+The API reference documentation is available in the [maistra/api repository](https://github.com/maistra/api/docs/crd/CRDS.adoc).
```
Comment (shamsher31) [inline README.md]: This link is broken
https://github.com/maistra/api/docs/crd/CRDS.adoc

Comment (shamsher31) [reply README.md]: It should be the following?
https://github.com/maistra/api/blob/maistra-2.1/docs/crd/CRDS.adoc

Comment (luksa) [reply README.md]: Thanks. I clicked on the initial link, but figured it was 404 because the api PR hadn't been merged yet.

**Review by shamsher31 — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Pull request merged**